src/app/templates/partials/metadata-tab.ftl

```diff
@@ -18,4 +18,27 @@
             ${elapsedString}
         </div>
     </div>
+    <#if report.gitBranch?? || report.gitCommit??>
```

Comment:
In practice we wouldn't expect to find a report version with one git value set but not the other (you'd expect both or none, depending on whether git is supported in the environment). But theoretically they could come out of the database that way, so coding the template as if they could be independent.

Reply:
When I tested this via new report run page I got a commit but not branch i.e. `git_branch` is empty in `report_version`. Is that what you would expect?

The branch is stored correctly in `orderlyweb_report_run` - which already looks like unfortunate redundancy (and probably my responsibility).

Reply:
Hm! Yeah, that's not quite what I was expecting. The demo db is misleading.. My theory was that maybe master branch (or default branch) doesn't get recorded but I tried pushing it up to UAT, and I can see there are some reports there with no branch recorded, and some with 'master'... I'll do a bit of digging..

Reply:
I now understand why git branch isn't getting recorded in the report_version table in reports we've run. Of course it's because we only pass the git commit as the ref param to orderly server. Other clients may pass a branch name and they will get both git branch and commit recorded. So as things stand, I will need to pull git branch from the orderly_web_report_run table after all. But I'll mention it to Rob and Rich as a bit of a blindspot in Orderlys record keeping!